packages/ui/src/components/bible-reader.tsx

```diff
@@ -87,6 +90,23 @@ function Root({
     onChange: onVersionChange,
   });
 
+  const [currentFontSize, setCurrentFontSize] = useState(() => {
+    if (typeof window === 'undefined') return fontSize;
+    const size = localStorage.getItem('font-size');
+    return size ? parseInt(size) : fontSize;
+  });
+
+  const [currentFontFamily, setCurrentFontFamily] = useState(() => {
+    if (typeof window === 'undefined') return fontFamily;
+    return localStorage.getItem('font-family') || fontFamily;
```

Comment:
Should we namespace this font setting? `youversion-platform:reader:font-family` or something like that?